aana/projects/chat_with_video/endpoints.py

```diff
@@ -154,6 +154,18 @@ async def run(
         if check_media_id_exist(media_id):
             raise MediaIdAlreadyExistsException(table_name="media", media_id=video)
 
+        video_duration = None
+        if video.url is not None:
+            video_info = get_video_metadata(video.url)
```

Comment:
What if video.url isn't a youtube link but a regular URL?

Reply:
Please add a test for that.

Reply:
done